docs/README.md

```diff
@@ -118,22 +118,27 @@ Booster's ultimate goal is making developer's lives easier, fulfilling the dream
 
 ### Booster Principles
 
-Booster takes a holistic and highly-opinionated approach at many levels:
+Booster enhances developers' productivity by focusing only on business logic. Write your code, provide your credentials and let Booster do the rest. Booster takes a holistic and highly-opinionated approach at many levels:
 
```

Comment:
I feel like the transition between those two sentences is not natural for me,, but right now I can't think of a better way to do it, so I guess we can leave it like this.

Reply:
OK @Moneyba . I checked it with Edith and it's grammatically correct. Maybe you are right but I don't know what to put instead